src/EventListener/FormListener.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Bolt\EventListener;
```

Comment:
Not sure I agree… It's a namespace that can/will grow… unless we make the namespace `Bolt\KernelEventListener` then it's make more sense.